src/ui/pages/codi/zcl_abapgit_gui_page_whereused.clas.abap

```diff
@@ -0,0 +1,222 @@
+CLASS zcl_abapgit_gui_page_whereused DEFINITION
+  PUBLIC
+  FINAL
+  INHERITING FROM zcl_abapgit_gui_component
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+    INTERFACES:
+      zif_abapgit_gui_page_title,
+      zif_abapgit_gui_event_handler,
+      zif_abapgit_gui_hotkeys,
+      zif_abapgit_gui_menu_provider,
+      zif_abapgit_gui_renderable,
+      zif_abapgit_html_table.
+
+    CLASS-METHODS create
+      IMPORTING
+        iv_package     TYPE devclass
+      RETURNING
+        VALUE(ri_page) TYPE REF TO zif_abapgit_gui_renderable
+      RAISING
+        zcx_abapgit_exception.
+
+    METHODS constructor
+      IMPORTING
+        iv_package TYPE devclass
+      RAISING
+        zcx_abapgit_exception.
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+    CONSTANTS:
+      BEGIN OF c_action,
+        refresh TYPE string VALUE 'refresh',
+      END OF c_action.
+
+    CONSTANTS c_title TYPE string VALUE 'Where Used'.
+    DATA mv_package TYPE devclass.
+    DATA mi_table TYPE REF TO zcl_abapgit_html_table.
+
+    METHODS init_table_component
+      RAISING
+        zcx_abapgit_exception.
+
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_GUI_PAGE_WHEREUSED IMPLEMENTATION.
+
+
+  METHOD constructor.
+    super->constructor( ).
+    mv_package = iv_package.
+
+    IF zcl_abapgit_factory=>get_sap_package( iv_package )->exists( ) = abap_false.
+      zcx_abapgit_exception=>raise( |Package { iv_package } does not exist| ).
+    ENDIF.
+
+  ENDMETHOD.
+
+
+  METHOD create.
+
+    DATA lo_component TYPE REF TO zcl_abapgit_gui_page_whereused.
+
+    CREATE OBJECT lo_component
+      EXPORTING
+        iv_package = iv_package.
+
+    ri_page = zcl_abapgit_gui_page_hoc=>create( lo_component ).
+
+  ENDMETHOD.
+
+
+  METHOD init_table_component.
+
+    DATA ls_sorting_state TYPE zif_abapgit_html_table=>ty_sorting_state.
+
+    IF mi_table IS BOUND.
+      RETURN.
+    ENDIF.
+
+    IF ls_sorting_state-column_id IS INITIAL.
+      ls_sorting_state-column_id = 'package'.
+    ENDIF.
+
+    mi_table = zcl_abapgit_html_table=>create(
+      ii_renderer              = me
+      is_initial_sorting_state = ls_sorting_state ).
+    mi_table->define_column_group(
+        iv_group_title = 'Repo object'
+        iv_group_id    = '' " No need
+      )->define_column(
+        iv_column_id    = 'dep_package'
+        iv_column_title = 'Pkg'
+      )->define_column(
+        iv_column_id    = 'dep_obj_type'
+        iv_column_title = 'Type'
+      )->define_column(
+        iv_column_id    = 'dep_obj_name'
+        iv_column_title = 'Obj name'
+      )->define_column(
+        iv_column_id    = 'dep_used_obj'
+        iv_column_title = 'Used obj' ).
```

Comment:
Maybe "Used Data Type" is better

Reply:
Hmm, are you sure there are only data types? (I'm just not sure exactly how it works, but in a much larger repo this field contains transactions too)

Reply:
It looks like the ABAP-internal representation that you also see in debugger. I would go for "ABAP Type" as in the [docs](https://help.sap.com/doc/abapdocu_latest_index_htm/latest/en-US/index.htm?file=abentype_hierarchy.htm)

Reply:
![image](https://github.com/abapGit/abapGit/assets/15635498/96341388-42a0-4c88-bff7-338c937a1123)

I'd keep as is ...